### ISSUE TITLE
Remove spare files

### ISSUE THREAD

**Issue by Rodrigap** [willgdq/willsjammmingapp#1]
https://github.com/willgdq/willsjammmingapp/blob/1512d5d6bce09f3c5f15531845bc81dbea5edcc9/src/App.js#L1-L21

I see you have this extra App.js, I know you're not using it, but it's much better practice to remove this file! Same with the extra Spotify.js file

**Comment by willgdq** [comment]
Good call!  Yes, when I moved to ../src/components/App I forgot to remove the older file.  Thanks!
